README.md

```diff
@@ -11,16 +11,39 @@ Snyk helps you find, fix and monitor for known vulnerabilities in your dependenc
 
 This plugin provides dependency metadata for Docker images.
 
-## Running Tests
+## Supported functionality
 
-To run tests the following environment variables need to be set:
+Package managers:
 
-`DOCKER_HUB_PRIVATE_IMAGE`
-`DOCKER_HUB_USERNAME`
-`DOCKER_HUB_PASSWORD`
+- rpm, apk, deb
 
-`DOCKER_HUB_PRIVATE_IMAGE` should refer to an image that is hosted on Docker Hub but not available publicly. During CI test this is set to `snykgoof/dockergoof:alpine`.
+Operating systems:
 
-## Writing Tests
+- Debian, Red Hat, Alpine, Oracle, CentOS, vanilla Linux, SLES
```

Comment:
A few things I'm worried about here
1. we will never remember to update it 😆 once we add new OSs
2. there are maybe others that we support? like Amazon Linux for example
3. there's a different between what the SDP supports and what Snyk supports. so for example the plugin also supports OpenSuse but Snyk does not

Reply:
It's on us to keep the documentation up to date 🙂  similarly to how we keep our code and tests, they should be an inseparable part of our work. And also up until now we didn't have any reference as to what we support - it's either in your head or you had to look up the code to figure it out.

For the others we support I will add them to the list, I think this is related to what the _plugin_ supports and not what Snyk does.